src/fetch_foursquare_info.py

```diff
@@ -0,0 +1,129 @@
+import os.path
+import requests
+import numpy as np
+import pandas as pd
+import re
+import json
+import datetime
+
+# Setting dataset paths
+REIMBURSEMENTS_DATASET_PATH = os.path.join('data', '2016-11-22-reimbursements.xz')
```

Comment:
Maybe we could dynamically load the most recent `YYYY-MM-DD-reimbursements.xz` instead of a fixed date (the same for `COMPANIES_DATASET_PATH`).